plugins/coral-plugin-love/client/LoveButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Icon} from 'coral-ui';
```

Comment:
@jde Do you consider this public api? What is our policy. Maybe only things that are explicitly documented in our repository is public api?